src/index.html

```diff
@@ -9,5 +9,29 @@
     <h1>Hello Push</h1>
     <img src="./assets/img/moramia.png" alt="image2">
     <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea rem, placeat assumenda facere voluptatem amet molestias tempora reiciendis quibusdam non, eaque dolorem natus labore ut, magni facilis itaque! Blanditiis, error?</p>
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-image">
+                <img src="./assets/img/logo2.png" alt="logo2" class="footer-img" />
+                <p class="footer-cp">
+                    Copyright ©2020 All rights reserved | This template is made with  &#9829;  by PushDev & Re-distributed by Moramia
+                </p>
+            </div>
+            <div class="menu-wrap">
+                <input type="checkbox" class="toggler">
+                <div class="hamburger"><div class="footer-center-line"></div></div>
```

Comment:
```suggestion
                <div class="hamburger"></div>
```
use a pseudo element in hamburger so you don't need to create an empty div element.

Reply:
I'm using pseudo elements for the top and bottom lines,  I  think that I need the middle one to use them, thats why I created that div

Reply:
ok but then do:
```suggestion
                <div class="hamburger"><span class="footer-center-line" role="presentation"></span></div>
```